lib/offline/OfflineStorage_SQLite.cpp

```diff
@@ -17,6 +17,9 @@ namespace MAT_NS_BEGIN {
 
     constexpr static size_t kBlockSize = 8192;
 
+    std::mutex OfflineStorage_SQLite::m_initAndShutdownLock;
+    int OfflineStorage_SQLite::m_instanceCount = 0;
```

Comment:
Should we make it `atomic` ?

Reply:
There's no need to, since there's already a lock, which is needed to prevent another initialization happening during shutdown (since shutdwon isn't thread safe for sqlite).